APIKit/APIKit.swift

```diff
@@ -77,11 +148,13 @@ public class API {
 
     // send request and build response object
     public class func sendRequest<T: Request>(request: T, handler: (Result<T.Response, NSError>) -> Void = {r in}) -> NSURLSessionDataTask? {
-        let session = URLSession()
+        let session = URLSession
         let mainQueue = dispatch_get_main_queue()
         
         if let URLRequest = request.URLRequest {
-            let task = session.dataTaskWithRequest(URLRequest) { data, URLResponse, connectionError in
+            let task = session.dataTaskWithRequest(URLRequest)
+            
+            task.completionHandler = { data, URLResponse, connectionError in
```

Comment:
When calling this handler, `dataTask.responseBuffer` isn't optional, so you can change `data` to non-optional :wink: